autokeras/task.py

```diff
@@ -203,6 +205,11 @@ def fit(self,
             y: String, numpy.ndarray, or tensorflow.Dataset. Training data y.
                 If the data is from a csv file, it should be a string corresponding
                 to the label column.
+            epochs: Int. The number of epochs to train each model during the search.
+                If unspecified, we would use epochs equal to 1000 and early stopping
```

Comment:
"If unspecified, by default we train for a maximum of 1000 epochs but we stop training if the validation loss stops improving for 30 epochs (unless you specified an `EarlyStopping` callback as part of the `callbacks` argument, in which case this callback will determine early stopping)."